spec/services/stream_daily_averages/fixed_active_sessions_traverser_spec.rb

```diff
@@ -17,10 +17,23 @@
       _mobile_stream = create_stream!(session: _mobile_session)
 
       expect(stream_interactor).to receive(:call).with(
-        { stream_id: stream_1.id, time_zone: time_zone },
+        { stream_id: stream_1.id, time_zone: time_zone, is_air_now_stream: false},
       )
       expect(stream_interactor).to receive(:call).with(
-        { stream_id: stream_2.id, time_zone: time_zone },
+        { stream_id: stream_2.id, time_zone: time_zone, is_air_now_stream: false},
+      )
+
+      subject.call
+    end
+
+    it 'traverses over AirNow streams and calls streams interactor for associated stream' do
+      time_zone = 'Europe/Warsaw'
+      user = User.create!(email: 'airnow@example.com', username: 'US EPA AirNow', password: 'password')
+      fixed_session = create_session!(type: 'FixedSession', time_zone: time_zone, user: user)
+      stream = create_stream!(session: fixed_session)
+
+      expect(stream_interactor).to receive(:call).with(
+        {stream_id: stream.id, time_zone: time_zone, is_air_now_stream: true}
```

Comment:
I would add `context` block here to clearly indicate that there is another case for AirNow stream.